internal/local/data.go

```diff
@@ -46,6 +46,7 @@ const (
 	NameFunctions   = "functions"
 	nameNodeModules = "node_modules"
 	NameSource      = "source"
+	nameJSON        = "package*.json"
```

Comment:
[q] why do we need the Kleene star here? is this to include `package-lock.json` files too?

Reply:
I had the `*` here because the test file could have name like `package-test.json`. But give it a second thought I just removed it.